data/tlg0011/tlg002/tlg0011.tlg002.perseus-grc2.xml

```diff
@@ -29,15 +28,22 @@
                 <biblStruct>
                     <monogr>
                         <author>Sophocles</author>
-                        <title>Sophocles. Vol 1: Oedipus the king. Oedipus at Colonus. Antigone.
-                            With an English translation by F. Storr. The Loeb classical library, 20. </title>
+                        <title>Sophocles</title>
+                        <title type="sub">Oedipus the king. Oedipus at Colonus. Antigone</title>
                         <editor>Francis Storr</editor>
                         <imprint>
-                            <pubPlace>London; New York</pubPlace>
-                            <publisher>William Heinemann Ltd.; The Macmillan Company</publisher>
+                            <pubPlace>London</pubPlace>
+                            <pubPlace>New York</pubPlace>
+                            <publisher>William Heinemann Ltd.</publisher>
+                            <publisher>The Macmillan Company</publisher>
                             <date>1912</date>
                         </imprint>
+                        <biblScope unit="volume">1</biblScope>
                     </monogr>
+                    <series xml:lang="lat">
```

Comment:
This got tagged as Latin throughout (may not have flagged all)

Reply:
The language stuff is so easy to miss in a larger PR. There are so many places to check. I'm sure I missed something.